src/index.ts

```diff
@@ -242,7 +267,8 @@ const fetchAX = {
           method: 'GET',
         });
 
-        if (requestArgs?.throwError) httpErrorHandling(response);
+        if (requestArgs?.throwError || isHttpError(response))
```

Comment:
```suggestion
        if (requestArgs?.throwError && isHttpError(response))
```
`throwError`이 `false`인 경우를 생각하면 `&&` 연산자의 사용이 적절해보여요

Reply:
그럼 throwError가 false이면 error interceptor가 실행 안되는 걸로 예상하신게 맞을까요?

Reply:
가현이 질문을 듣고 고민해보니 `throwError`와 `error interceptor`의 실행이 독립적여야한다는 생각이 드네요. 다만 한가지 궁금한 점은, `throwError`가 false인 경우에도 에러가 300이상의 state를 반환하는 경우에는 `Promise.reject(error)`가 return될 것으로 예측되는데 이 부분이 문제가 되지는 않을까요?

Reply:
제 생각에는 throwError가 true일 때만 에러를 반환하는 것이 맞다는 생각이 드네요. 이에 따라 수정했습니다.

Reply:
3c6962675ea4b91da4e10962b87d1b3ad42eea0a